output-tests/README.md

```diff
@@ -0,0 +1,30 @@
+These tests are intended to validate that implementations are correctly generating output in accordance with the specification.
+
+Output was initially specified with draft 2019-09.  It remained unchanged for draft 2020-12, but will receive an update with the next release.
```

Comment:
Didn't we change "instanceLocation" from a URL to jsut a JSON Pointer in 2020-12?

Reply:
I thought it was always a pointer; just that I was rendering it as a URI fragment in my implementation (and probably in the spec examples).

Reply:
From 2019-09, https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-02#section-10.3.3

> The location of the JSON value within the instance being validated. The value MUST be expressed as a URI fragment-encoded JSON Pointer.

From 2020-12, https://json-schema.org/draft/2020-12/json-schema-core.html#section-12.3.3

> The location of the JSON value within the instance being validated. The value MUST be expressed as a JSON Pointer.

Reply:
Maybe "It remained _largely_ unchanged..."?

Reply:
That works for me